brlaser.drv.in

```diff
@@ -334,6 +334,13 @@ Option "brlaserEconomode/Toner save mode" Boolean AnySetup 10
   PCFileName "br2140.ppd"
 }
 
+{
+  ModelName "HL-2150N"
+  Attribute "NickName" "" "Brother HL-2150N, $USING"
+  Attribute "1284DeviceID" "" "MFG:Brother;CMD:PJL,HBP;MDL:HL-2150N;CLS:PRINTER;"
+  PCFileName "br2150n.ppd"
```

Comment:
The `PCFileName` should be `br2150.ppd`.

We don't use `n` or `w` unless there is a conflict.

Reply:
Thank you for the review :+1: Fixed it